<launch>
    <node  pkg="rrr_arm" type="rrr_arm_server.py" name="task_server" respawn="True" output="screen" />

    <node pkg="rrr_arm" type="alexa_interface.py" name="alexa_interface" respawn="True" output="screen" />
</launch>